Makefile

```diff
@@ -1,11 +1,12 @@
 IN_ENV = if [ -e .venv/bin/activate ]; then . .venv/bin/activate; fi;
 
 
-install: packages venv python_packages install_fasm2bels install_yosys
+install: packages install_wafove venv python_packages install_fasm2bels install_yosys
 
 venv:
 	python3 -m venv .venv
 	$(IN_ENV) python3 -m pip install -U pip
+	$(IN_ENV) python3 -m pip install -e third_party/WaFoVe
```

Comment:
this is probably the wrong place for `pip install`ing your package.  If you decide that `pip` is the way to go, it should either be added to the `requirements.txt` to be picked up by the `python_packages` target, or the `pip install` should be a part of the `install_wafove` target.